src/input/InputComponent.cpp

```diff
@@ -166,9 +176,6 @@ void InputComponent::remapInput(const QString &source, const QString &keycode, b
     return;
   }
 
-  // hide mouse if it's visible.
-  SystemComponent::Get().setCursorVisibility(false);
-
```

Comment:
Oh I see, it got moved in the previous commit. These should probably be squashed.